tests/fast-integration/eventing-test/common/common.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+const eventMeshSecretFilePath = process.env.EVENTMESH_SECRET_FILE || '';
+const eventMeshNamespace = getEventMeshNamespace();
+const natsBackend = 'nats';
+const bebBackend = 'beb';
+
+// returns the EventMesh namespace from the secret.
+function getEventMeshNamespace() {
+  try {
+    const eventMeshSecret = JSON.parse(fs.readFileSync(eventMeshSecretFilePath, {encoding: 'utf8'}));
+    return '/' + eventMeshSecret['namespace'];
+  } catch (e) {
+    console.error(e);
+    return undefined;
```

Comment:
(nit) Should we throw the error, so we don't continue the tests if we fail here for beb?